.eslintrc.js

```diff
@@ -99,6 +99,41 @@ module.exports = {
         'react/require-default-props': 'off',
       },
     },
+    // Rules we want to enforce or change for Stimulus Controllers
+    {
+      files: ['*Controller.ts'],
+      rules: {
+        '@typescript-eslint/member-ordering': [
+          'error',
+          {
+            classes: {
+              memberTypes: ['constructor', 'field', 'method', 'signature'],
```

Comment:
good point - I mainly just copied this from the docs for this (I thought some order is better than no order).

I think there could be small edge cases where we may still want constructor and if we pull this eslint rule up to all classes in code it would be nice to have the same approach.